nixos/modules/services/monitoring/certspotter.nix

```diff
@@ -133,6 +133,8 @@ in
       serviceConfig = {
         User = "certspotter";
         Group = "certspotter";
+        Restart = "on-failure";
+        RestartSec = "10s";
```

Comment:
I'd suggest a longer duration (a couple of minutes maybe?), to minimize logspam if not for any other reason. Failing that, one can set it up so that the rate of retries slows down (using `RestartSteps` and `RestartMaxDelaySec`).

Reply:
restarting is a cheap operation in this case, and ~10 second delays are very common (just `grep Restart nixpkgs`)